<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arbeitsdatenbank - Projektsuche</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1" columnminimumwidth="250,0">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="navbar_verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="titel_lineEdit">
        <property name="placeholderText">
         <string>Titel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stichwort_lineEdit">
        <property name="placeholderText">
         <string>Stichworte</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QComboBox" name="betreuer_comboBox">
          <property name="placeholderText">
           <string>Betreuer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_betreuer_pushButton">
          <property name="text">
           <string>üóëÔ∏è</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="von_lineEdit">
          <property name="placeholderText">
           <string>Von</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bis_lineEdit">
          <property name="placeholderText">
           <string>Bis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="bearbeiter_lineEdit">
        <property name="placeholderText">
         <string>Bearbeiter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="dozentensuche_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Dozentensuche</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="projektsuche_pushButton">
        <property name="text">
         <string>Projektsuche</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dozent_erstellen_pushButton">
        <property name="text">
         <string>Dozent erstellen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arbeit_erstellen_pushButton">
        <property name="text">
         <string>Arbeit erstellen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="account_bearbeiten_pushButton">
        <property name="text">
         <string>Angemeldeten Account bearbeiten</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logout_pushButton">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="login_pushButton">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="content_verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="arbeiten_listWidget"/>
      </item>
      <item>
       <widget class="QListWidget" name="dozenten_listWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
